src/garage/tf/regressors/bernoulli_mlp_regressor.py

```diff
@@ -153,12 +153,10 @@ def _initialize(self):
 
             self._optimizer.update_opt(loss=loss,
                                        target=self,
-                                       network_output=[y_hat],
```

Comment:
Removed it since none of the `garage.tf.optimizers` uses this. It was causing some regressor tests to fail since I removed `*kwargs` from `ConjugateGradientOptimizer.update_opt`.